app/assets/stylesheets/searchbar.scss

```diff
@@ -0,0 +1,48 @@
+
+/* ---------- SEARCH ---------- */
+
+.search-box {
+  text-align: center;
+  margin-top: 60px;
+  padding: 0;
+}
+
+#search {
+	background: transparent;
+	border-radius: 3px;
+	display: inline-block;
+	padding: 7px;
+}
+
+input#q.aaa-search-input {
+  width: 200px;
+  height: 50px;
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+}
+
+.aaa-search-btn {
+  height: 50px;
+  background: #1bba9a;
+  border-top-right-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+
+.aaa-search-btn:hover {
+  background: #189e83;
+}
+
+#search input {
+	float: left;
+}
+
+#search input[type="search"] {
+	background: #fff;
+	color: #42454e;
+	min-width: 184px;
+	padding: 6px 8px;
+}
+
+#search input[type="search"]::-webkit-input-placeholder { color: #42454e; }
+#search input[type="search"]:-moz-placeholder { color: #42454e; }
+#search input[type="search"]:-ms-input-placeholder { color: #42454e; }
```

Comment:
Color literals like `#42454e` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Avoid using id selectors<br>Merge rule `#search input[type="search"]:-ms-input-placeholder` with rule on line 10<br>Avoid qualifying attribute selectors with an element.